install_config/certificate_customization.adoc

```diff
@@ -284,6 +284,65 @@ openshift_hosted_registry_routetermination=reencrypt <3>
 +
 * Set to `passthrough` to terminate encryption at the destination. The destination is responsible for decrypting traffic.
 
+[[configuring-custom-certificates-lb]]
+== Configuring a Custom Certificate for a Load Balancer
+
+If your {product-title} cluster uses the default load balancer or an enterprise-level load balancer, you can use custom certificates
+to make the web console & API available externally using a publicly-signed custom certificate and leave the internal endpoints to use the existing internal certificates.
+
+//The load balancer terminates SSL connections for incoming traffic, then re-encrpyts and sends the traffic to the masters. 
+
+To configure {product-title} to use custom certificates in this way:
+
+. Edit the `servingInfo` section of the xref:../install_config/master_node_configuration.adoc#master-configuration-files[master configuration file]:
++
+----
+servingInfo:
+  logoutURL: ""
+  masterPublicURL: https://openshift.example.com:8443
+  publicURL: https://openshift.example.com:8443/console/
+  namedCertificates:
+    bindAddress: 0.0.0.0:8443
+    bindNetwork: tcp4
+    certFile: master.server.crt (1)
+    clientCA: ""
+    keyFile: master.server.key (1)
+    maxRequestsInFlight: 0
+    requestTimeoutSeconds: 0
+    namedCertificates:
+      - certFile: wildcard.example.com.crt (2)
+        keyFile: wildcard.example.com.key (2)
+        names:
+          - "openshift.example.com"
+  metricsPublicURL: "https://metrics.os.example.com/hawkular/metrics"
+----
++
+<1> Path to the certificate and key files for the CLI and other API calls.
++
+<2> Path to the certificate and key files for the web console.
++
```

Comment:
According to my knowledge, <1> is irrelevant to this topic, no need highlight it here, suggest to remove it.

Reply:
@jianlinliu  I removed the <1> and split the <2> Path to the certificate and key files... line.

<1> Path to the certificate files for the web console.
<2> Path to the key files for the web console.